### ISSUE TITLE
Technology Stack

### ISSUE THREAD

**Issue by thasup** [peuan/travelmate-web#3]
## TL/DR

- Purpose: Select the appropriate technology stack for our _travelmate_ web application.

#### Core stack
- ⚡️ [Next.js 13](https://nextjs.org/) - Front-end Framework
- 🦾 [TypeScript](https://www.typescriptlang.org/) - Front-end programming language
- 🎨 [MUI](https://mui.com/) - UI components

#### Dependencies
- ⚙️ [SWR](https://swr.vercel.app/) - React Hook for Data Fetching
- ⚙️ [Axios](https://axios-http.com/docs/intro) - HTTP Client library
- ⚙️ [Yarn](https://yarnpkg.com/) - Package Manager
- ⚡️ [Jest](https://jestjs.io/) - Unit Testing Framework
- ⚡️ [Playwright](https://playwright.dev/) - E2E Testing Framework
- 🎨 [ESlint](https://eslint.org/) - Linter for JavaScript and TypeScript
- 🎨 [Prettier](https://prettier.io/) - Code Formatter
- 🎨 [Stylelint](https://stylelint.io/) - Linter for styles
---

#### Core stack
- ⚡️ [Nest.js](https://nestjs.com/) - Back-end Framework
- ⚡️ [Node.js](https://nodejs.org/en/) - Javascript Runtime
- 🦾 [TypeScript](https://www.typescriptlang.org/) - Back-end programming language
- ⚡️ [MySQL](https://www.mysql.com/) - Database

#### Dependencies
- 📦 [TypeORM](https://typeorm.io/) - Object-Relational Mapping library
- ⚡️ [Jest](https://jestjs.io/) - Testing Framework
- 🎨 [ESlint](https://eslint.org/) - Linter for JavaScript and TypeScript
---

Please feel free to suggest and discuss any stack you would like to explore. I am confident that we all enjoy discovering interesting stacks in this project!
